src/workers/data-importer.ts

```diff
@@ -76,47 +80,56 @@ export class BundleDataImporter {
   }
 
   async queueItem(
-    item: UnbundleableItem,
+    item: ImportableItem,
     prioritized: boolean | undefined,
     bypassFilter = false,
   ): Promise<void> {
     const log = this.log.child({ method: 'queueItem', id: item.id });
     if (this.workerCount === 0) {
-      log.debug('Skipping bundle download, no workers.');
+      log.debug('Skipping contiguous-data download, no workers.');
       return;
     }
 
     if (prioritized === true) {
-      log.debug('Queueing prioritized bundle download...');
+      log.debug('Queueing prioritized contiguous-data download...');
       this.queue.unshift({ item, prioritized, bypassFilter });
-      log.debug('Prioritized bundle download queued.');
+      log.debug('Prioritized contiguous-data download queued.');
     } else if (this.queue.length() < this.maxQueueSize) {
-      log.debug('Queueing bundle download...');
+      log.debug('Queueing contiguous-data download...');
       this.queue.push({ item, prioritized, bypassFilter });
-      log.debug('Bundle download queued.');
+      log.debug('contiguous-data download queued.');
     } else {
-      log.debug('Skipping bundle download, queue is full.');
+      log.debug('Skipping contiguous-data download, queue is full.');
     }
   }
 
   async download({
     item,
     prioritized,
     bypassFilter,
-  }: UnbundlingQueueItem): Promise<void> {
+  }: DataImporterQueueItem): Promise<void> {
     const log = this.log.child({ method: 'download', id: item.id });
 
     const data = await this.contiguousDataSource.getData({ id: item.id });
 
     return new Promise((resolve, reject) => {
       data.stream.on('end', () => {
-        log.debug('Bundle data downloaded complete. Queuing for unbundling..');
-        this.ans104Unbundler.queueItem(item, prioritized, bypassFilter);
+        const isAnyContiguousData = this.isAnyContiguousData(item);
```

Comment:
What is the purpose of `isAnyContiguousData`?

Reply:
in this case the item passed has just { id: string; } it's not the full data-item as we have in many places. I made a type for the two variants. From the verification side we don't have this data-item interface and just the tx-id. And the data-importer class cannot accept import data in that form as is. So it's a suggestion of how to solve it, and here isAnyContiguousData is just asserting that it's this { id: string; } format, which can be either layer-1 or layer-2 (maybe it means it's just layer-1 only because I think we only verify layer-1 data in the dataRootVerify, so a better variable name may be needed)

Reply:
I'm reversing this logic, it should ask for isUnbundleableItem or not, that's an interface that's already present there.

Reply:
done